Add new spreadsheet workouts (#2140)
enkidevs/curriculum: spreadsheets/intermediate/charts-2/example-use-case-for-charts.md

---
author: Stefan-Stojanovic

aspects:

>   - introduction

Reviewer: since we're in the intermediate topic, I don't think the `introduction` aspect is fitted here

  - workout

type: normal

category: how to

---

# Example use case

---
## Content

Charts are a great way of visualizing data.

> Let's say we have a table spanning 3 years where there is an `Employee of the month` for each month of the 3 years.

Reviewer: what do you think about this?
```suggestion
Let's say we have a table spanning over 3 years, with an `Employee of the month` for each month.
```

![table](https://img.enkipro.com/1f584d886ee197028c667f42f509ef2c.png)

Since our table has only 36 entries it wouldn't be to hard to achieve our goal without a chart.

> However, we can use a chart to visualize and see who was the `Employee of the month` the most out of the 3 years in just a few steps.

Reviewer: ```suggestion
However, we can use a chart to visualize and see who got `Employee of the month` the most, in just a few steps.
```

- select all the cells you want to include in your chart
- go to `insert - > chart`
- select the chart you want
- tick the `aggregate` button in the chart

Chart:

![chart](https://img.enkipro.com/70eca1f472c36ead8ece0c5342bb767d.png)

> As you can see in the image above, it is much easier to visualize the data into a chart and then determine who has been the `Employee of the month` for most months. 

Reviewer: ```suggestion
Now, it's much easier to visualize the data and then determine who has been `Employee of the month` the most. 

```
